runtime/themes/tokyonight.toml

```diff
@@ -41,6 +41,9 @@
 "ui.selection.primary" = { bg = "background_highlight" }
 "ui.statusline" = { fg = "foreground", bg = "background_menu" }
 "ui.statusline.inactive" = { fg = "foreground_gutter", bg = "background_menu" }
+"ui.statusline.normal" = { fg = "black", bg = "blue" }
+"ui.statusline.insert" = { fg = "black", bg = "green" }
+"ui.statusline.select" = { fg = "black", bg = "magenta" }
```

Comment:
Instead of 'dark' for the color, I think background_gutter makes more sense.
This is a very weak opinion tho, as it doesn't really change anything in how it looks.

Reply:
Hmm, `background_gutter` is too light:
<img width="654" alt="image" src="https://user-images.githubusercontent.com/2804556/197355366-703802b4-af61-43ae-ac31-c5831b78821d.png">

But I see you had `background_menu` in your PR, could that be the color you meant?

That said, the reason I initially went with `black` was because, if we ever create a light theme version of tokyonight (such as here: https://github.com/folke/tokyonight.nvim#day), it would be important that the foreground color remains black/dark, regardless of the gutter/menu background colors (which would most definitely become light).

What do you think?

Reply:
> Hmm, `background_gutter` is too light: <img alt="image" width="654" src="https://user-images.githubusercontent.com/2804556/197355366-703802b4-af61-43ae-ac31-c5831b78821d.png">
> 
> But I see you had `background_menu` in your PR, could that be the color you meant?
> 
> That said, the reason I initially went with `black` was because, if we ever create a light theme version of tokyonight (such as here: https://github.com/folke/tokyonight.nvim#day), it would be important that the foreground color remains black/dark, regardless of the gutter/menu background colors (which would most definitely become light).
> 
> What do you think?

That's a decent enough reason. Because i haven't thought about it, i just had it consistent with the inactive one.
Otherwise, if that one doesn't look good, i am fine with the changes. :) Good job! :)